src/rust/engine/testutil/mock/src/execution_server.rs

```diff
@@ -127,7 +128,7 @@ impl Drop for TestServer {
 #[derive(Clone, Debug)]
 pub struct MockResponder {
   mock_execution: MockExecution,
-  received_messages: Arc<Mutex<Vec<(String, Box<protobuf::Message>)>>>,
+  pub received_messages: Arc<Mutex<Vec<(String, Box<protobuf::Message>, SystemTime)>>>,
```

Comment:
@illicitonion is it okay to make it `pub`? same as above

Reply:
Works for me :)